frontend/src/components/organisms/TableFilters/index.jsx

```diff
@@ -3,6 +3,7 @@ import "react-calendar/dist/Calendar.css";
 import "./index.less";
 
 import {
+  Banner,
```

Comment:
You added the `Banner` import, but I don't see it being used. Ah, well. I see you merged, so someone can clean it up later. I'll bet `ESLint` will catch it.